server/app.js

```diff
@@ -17,27 +18,38 @@ const app = express();
 /*
  * Check request
  */
-app.use(helper.handlerWrapper(async (req, _res, next) => {
-  if(req.method === "POST" || req.method === "PATCH"  || req.method === "PUT" ){
-    if(req.headers['content-type'] !== "application/json"){
-    throw new HttpError(415, "Invalid content type. API only supports application/json");
+app.use(
+  helper.handlerWrapper(async (req, _res, next) => {
+    if (
+      req.method === 'POST' ||
+      req.method === 'PATCH' ||
+      req.method === 'PUT'
+    ) {
+      if (req.headers['content-type'] !== 'application/json') {
+        throw new HttpError(
+          415,
+          'Invalid content type. API only supports application/json',
+        );
+      }
     }
-  }
-  next();
-}));
+    next();
+  }),
+);
 
-app.use(bodyParser.urlencoded({ extended: false })); // parse application/x-www-form-urlencoded
-app.use(bodyParser.json()); // parse application/json
+app.use(express.urlencoded({ extended: false })); // parse application/x-www-form-urlencoded
+app.use(express.json()); // parse application/json
 
 app.use('/captures', captureRouter);
+app.use('/raw-captures', rawCaptureRouter);
```

Comment:
Treetracker api only has `captures` and `trees` as resources. `raw-captures` is owned by field-data-service. So we need to remove `raw-captures` from here.